### PR TITLE
Gradle consistent versions

### PR DESCRIPTION
**Pull request by j-baker**
First stab at this. Unclear if it's actually going to work, I removed some important looking stuff.

### CODE REVIEW

**Comment by gmaretic** [comment]
This looks good to me, @jeremyk-91 to approve?

File: atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/NameMetadataDescription.java [hunk 1]
```diff
@@ -181,55 +174,6 @@ public String renderToJson(byte[] name) {
         return sb.toString();
     }
 
-    public byte[] parseFromJson(String json, boolean allowPrefix) {
```
Comment (jeremyk-91) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/NameMetadataDescription.java:184]: This is used by large internal product's command line interface to an AtlasDB database, so I'm not sure we can remove this?

**Comment by jeremyk-91** [comment]
I will pick up fixing the removed method this week.

File: atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/NameMetadataDescription.java (hunk 1 above)
Comment (jeremyk-91) [reply atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/NameMetadataDescription.java:184]: This deletion is not needed after #4821

**Comment by jeremyk-91** [comment]
Merging: build has passed, James wrote the PR and Grgur effectively approved it, I just resolved the merge conflicts.

**Pull request merged**